### PR TITLE
basic-features.md: add a column for v8.3.0

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -20,245 +20,245 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
 | [元数据锁](/metadata-lock.md) |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N |
-| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N |
-| [暂停](/sql-statements/sql-statement-admin-pause-ddl.md)/[恢复](/sql-statements/sql-statement-admin-resume-ddl.md) DDL |  Y  |  Y  | Y | N | N | N | N | N | N | N |
-| [TiDB 加速建表](/accelerated-table-creation.md) | E | E | N | N | N | N | N | N | N | N |
-| [设置 BDR Role 用于 TiCDC 双向同步时同步 DDL](/sql-statements/sql-statement-admin-bdr-role.md#admin-setshowunset-bdr-role) | E | E | N | N | N | N | N | N | N | N |
+| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) |  Y  |  Y  |  Y  |  Y  | Y | Y | N | N | N | N | N |
+| [暂停](/sql-statements/sql-statement-admin-pause-ddl.md)/[恢复](/sql-statements/sql-statement-admin-resume-ddl.md) DDL |  Y  |  Y  |  Y  | Y | N | N | N | N | N | N | N |
+| [TiDB 加速建表](/accelerated-table-creation.md) | E | E | E | N | N | N | N | N | N | N | N |
+| [设置 BDR Role 用于 TiCDC 双向同步时同步 DDL](/sql-statements/sql-statement-admin-bdr-role.md#admin-setshowunset-bdr-role) | E | E | E | N | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:134]:
```suggestion
| [设置 BDR Role 用于 TiCDC 双向同步时同步 DDL](/sql-statements/sql-statement-admin-bdr-role.md#admin-setshowunset-bdr-role) | Y | E | E | N | N | N | N | N | N | N | N |
```

**Review by Frank945946 — APPROVED** [review]

File: basic-features.md [hunk 2]
```diff
@@ -20,245 +20,245 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| SQL 语句 [^3] | 8.3 | 8.2 | 8.1 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| `SELECT`，`INSERT`，`UPDATE`，`DELETE`，`REPLACE` | Y | Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| `INSERT ON DUPLICATE KEY UPDATE` | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| `LOAD DATA INFILE` | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| `SELECT INTO OUTFILE` | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| `INNER JOIN`, <code>LEFT\|RIGHT [OUTER] JOIN</code> | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| `UNION`，`UNION ALL` | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [`EXCEPT` 和 `INTERSECT` 运算符](/functions-and-operators/set-operators.md) | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| `GROUP BY`，`ORDER BY` | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
```
Comment (Oreoxmt) [inline basic-features.md:77]:
```suggestion
| `GROUP BY`，`ORDER BY` | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
| [`GROUP BY` 修饰符](/functions-and-operators/group-by-modifier.md) | Y |  Y |  Y |  N  | N | N | N | N | N | N | N |
```

Comment (joey-yez) [reply basic-features.md:77]: 这里是不是要4个 Y？

File: basic-features.md [hunk 3]
```diff
@@ -20,245 +20,245 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [使用 bucket 提高数据扫描并发度](/tune-region-performance.md#使用-bucket-增加并发) | E |  E |  E |  E  | E | E | E | N | N | N | N |
+| [不可见索引](/sql-statements/sql-statement-create-index.md#不可见索引) | Y | Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [复合主键](/constraints.md#主键约束) | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [`CHECK` 约束](/constraints.md#check-约束) | Y |  Y |  Y |  Y  | N | N | N | N | N | N | N |
+| [唯一约束](/constraints.md#唯一约束) | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [整型主键上的聚簇索引](/clustered-indexes.md) | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [复合或非整型主键上的聚簇索引](/clustered-indexes.md) | Y |  Y |  Y |  Y  | Y | Y | Y | Y | Y | Y | Y |
+| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | Y |  Y |  Y |  Y  | Y | N | N | N | N | N | N |
+| [外键约束](/constraints.md#外键约束) | E | E |  E |  E  | E | N | N | N | N | N | N |
+| [TiFlash 延迟物化](/tiflash/tiflash-late-materialization.md) | Y |  Y |  Y |  Y  | Y | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:65]:
```suggestion
| [全局索引 (Global Index)](/partitioned-table.md#全局索引) | E |  N |  N | N  | N | N | N | N | N | N | N |
```

File: basic-features.md (hunk 2 above)
Comment (joey-yez) [reply basic-features.md:77]: LGTM

File: basic-features.md [hunk 4]
```diff
@@ -20,245 +20,246 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [显示统计信息收集的进度](/sql-statements/sql-statement-show-analyze-status.md) |  Y  |  Y  |  Y   | N | N | N | N | N | N | N |
+| 统计信息 | 8.3 | 8.2 | 8.1 | 7.5  | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 |
+|---|:----:|:----:|:----:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [CM-Sketch](/statistics.md) | 默认关闭| 默认关闭| 默认关闭| 默认关闭 | 默认关闭 | 默认关闭 | 默认关闭 | 默认关闭 | 默认关闭 | Y | Y |
+| [直方图](/statistics.md) |  Y  |  Y  |  Y  |  Y   | Y | Y | Y | Y | Y | Y | Y |
+| [扩展统计信息](/extended-statistics.md) |  E  |  E  |  E  |  E   | E | E | E | E | E | E | E |
+| 统计反馈 |  N  |  N  |  N  |  N   | N | N | 已废弃 | 已废弃 | E | E | E |
+| [统计信息自动更新](/statistics.md#自动更新) |  Y  |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y |
+| [动态裁剪](/partitioned-table.md#动态裁剪模式) |  Y  |  Y   |  Y   |  Y   | Y | Y | Y | E | E | E | E |
+| [收集部分列的统计信息](/statistics.md#收集部分列的统计信息) |  E   |  E   |  E   |  E   | E | E | E | E | N | N | N |
```
Comment (lilin90) [inline basic-features.md:179]:
```suggestion
| [收集部分列的统计信息](/statistics.md#收集部分列的统计信息) |  Y   |  E   |  E   |  E   | E | E | E | E | N | N | N |
```

File: basic-features.md (hunk 3 above)
Comment (Frank945946) [reply basic-features.md:65]: LGTM

**Review by Frank945946 — APPROVED** [review]

File: basic-features.md [hunk 5]
```diff
@@ -20,245 +20,246 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [DM](/dm/dm-overview.md) 支持迁移 MySQL 8.0  | Y | Y | E | E | E | E | N | N | N | N |
+| 数据导入和导出 | 8.3 | 8.2 | 8.1 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [快速导入 (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [快速导入 (`IMPORT INTO`)](/sql-statements/sql-statement-import-into.md) | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| mydumper 逻辑导出 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 |
+| [Dumpling 逻辑导出](/dumpling-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) [^5] | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [数据迁移工具](/migration-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) [^6] | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```
Comment (lilin90) [inline basic-features.md:215]:
```suggestion
| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) [^6] | 已废弃 | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```

**Review by qiancai — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**